metakb/version.py

```diff
@@ -1,4 +1,2 @@
 """MetaKB version"""
-# REQ: EACH TIME VERSION IS UPDATED, MUST ALSO UPDATE LAST_UPDATED
-__version__ = "1.1.0"
-LAST_UPDATED = "2022-11-08"
+__version__ = "2.0.0-dev0"
```

Comment:
WOW i was wondering about this

Reply:
This was an old thing that was needed for VarCat, but it's no longer being used